integration-tests/tests/pages/search.page.ts

```diff
@@ -27,9 +27,12 @@ export class SearchPage {
     }
 
     async enableSearchFields(...fieldLabels: string[]) {
```

Comment:
@theosanderson and @fhennig this subfunction is really flaky - it has been the cause of failing integration tests for most of this PR... I tried to now add another await but it doesn't seem to suffice - do you have any ideas? (When I run locally I see that the pop up is not opening although we navigate to the `'Add Search Fields'` and it looks like it is clicked)

Reply:
weirdly the tests only fail on chromium

Reply:
I mean what happens if one reverts the changes to this function? Or what are they needed for?

Reply:
It's not clear to me why we we need to wait for something to be visible, for example (that's normally inherent)